pkg/cmd/clustertask/start.go

```diff
@@ -162,6 +166,7 @@ like cat,foo,bar
 	c.Flags().StringVar(&opt.TimeOut, "timeout", "", "timeout for TaskRun")
 	c.Flags().BoolVarP(&opt.DryRun, "dry-run", "", false, "preview TaskRun without running it")
 	c.Flags().StringVarP(&opt.Output, "output", "", "", "format of TaskRun dry-run (yaml or json)")
+	c.Flags().BoolVarP(&opt.UseParamDefaults, "use-param-defaults", "", false, "use default parameter values without prompting for input for those parameters")
```

Comment:
Keep the description consistent with task/pipeline start. Also, since there is no shorthand, use `BoolVar`.

```suggestion
	c.Flags().BoolVar(&opt.UseParamDefaults, "use-param-defaults", false, "use default parameter values without prompting for input")
```

Reply:
Addressed :+1: